GOVERNANCE.md

```diff
@@ -225,6 +227,35 @@ Additionally, there is a strict process for assigning Project-Funded Bounties:
   - If an objection is raised, the request is put on hold until a Lead makes the final decision.
 - There is a maximum amount of issues with pledges someone can have assigned to them. The limit is 2 for core contributors, and 1 for anyone else. By limiting the amount of issues with pledges that someone can have assigned, we make sure the bounties remain available for others to pick up.
 
+### Paid Contracting
+
+Core Contributors may enter freelance contracts with clients to work on Biome. Such contracts are
+between the contributor and their client, so they fall mostly outside the responsibility of the
+Biome project. Nevertheless, we can explicitly endorse such contracts under the following
+conditions:
+
+- Core Contributors that are open to contracts may be advertised on the project website if they
+  please.
+- Clients that hire a Core Contributor for an extended period (3 months or more) are eligible to
+  the same benefits as a project sponsor. Their sponsorship benefits will be based on the monthly
+  fee paid to the Core Contributor.
+- If the work that is expected to be delivered does not contribute to Biome's last-published
+  roadmap, Biome asks for a 30% fee over the total amount earned through the contract. This is to
```

Comment:
Do we have any roadmaps published? Who decides what contributes to the roadmap? (vote, contributor discretion, etc)

Reply:
Yes, the last one is here: https://biomejs.dev/blog/roadmap-2024/
So we're due for an update :)

The process for defining the roadmap is in the `GOVERNANCE` doc itself, see the Roadmap section.

Reply:
> last-published roadmap

This implies that we have a roadmap, which forces us to have one at least once a year. It's actually a lot of work for us, and I'm not sure we should impose an additional responsibility on us. 

Instead, I would propose a different wording in which the core team checks whether the job is in line with Biome goals.

Reply:
I do think it's good to have a roadmap in general. Maybe I can help publish the next one next week :)

But yeah, maybe I can this: "If the last-published roadmap is more than 12 months old, the Core Contributor is asked to consult with the Core team about alignment with the project goals."

Reply:
I think that having a roadmap is good for the project, too. However, we have never mentioned the concept of "roadmap" in the governance, so we can't draft this new part of the governance (contracting) by mentioning something that isn't already in the governance. It's just a loophole 😅

Reply:
*cough* https://github.com/biomejs/biome/blob/main/GOVERNANCE.md#roadmap :stuck_out_tongue:

Reply:
LOL I completely forgot about it. Past Ema thought about everything lol

Reply:
Added sentence.